### DISCUSSION TITLE
我該如何學習

### DISCUSSION THREAD

**Discussion by Nijika1TAU** [Nijika1TAU/shell_LINUX#1] [Q&A] [answered]
幫幫忙

**Comment by sfsf100** [comment, marked as answer]
「程式筆記」Shell Script 簡易筆記
https://medium.com/@yihengwu/%E7%A8%8B%E5%BC%8F%E7%AD%86%E8%A8%98-shell-script-%E7%B0%A1%E6%98%93%E7%AD%86%E8%A8%98-841cfc3ae3ab

供您參考
